src/worldcereal/openeo/preprocessing.py

```diff
@@ -276,7 +280,17 @@ def worldcereal_preprocessed_inputs_gfmap(
     backend_context: BackendContext,
     spatial_extent: BoundingBoxExtent,
     temporal_extent: TemporalContext,
+    fetch_type: Optional[FetchType] = FetchType.TILE,
+    collections: Optional[List[str]] = None,
 ) -> DataCube:
+    if not all(
+        coll in {"sentinel1", "sentinel2", "copernicus", "meteo"}
```

Comment:
please use "cop_dem" or "dem" as collection name instead of "copernicus"

Reply:
Resolved by latest commit